### DISCUSSION TITLE
Rebind window mode and error log

### DISCUSSION THREAD

**Discussion by rozaliev** [helix-editor/helix#7318] [Q&A] [answered]
Hi, I'm trying to find a way to rebind `window mode` to another hotkey, is it possible? There is no `window_mode` command.

While experimenting, I've encountered errors like on the screenshot, but they quickly disappeared, how can I access a list of such errors? The diagnostic picker is empty and after installation of toml lsp (taplo) it just says `this document has been excluded`, which seems to be a problem in itself.

![Screenshot 2023-06-12 at 11 50 28](https://github.com/helix-editor/helix/assets/120809/489a31af-87f0-429d-9707-839d45a20b79)

**Comment by the-mikedavis** [comment, marked as answer]
Remapping groups of bindings is not currently possible (https://github.com/helix-editor/helix/issues/1508), you need to remap the individual commands within window mode.

Statusline errors are also written to the logfile: https://github.com/helix-editor/helix/wiki/FAQ#access-the-log-file

**Comment by madhavkhoslaa** [comment]
Still not possible :(
